.gitpod.yml

```diff
@@ -0,0 +1,6 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: 'echo "TODO: Replace with init/build command"'
+    command: 'echo "TODO: Replace with command to start project"'
```

Comment:
These commands don't seem very useful to me. Also, we don't currently have any setup steps to automate in this repository (no dependencies, no init, no project to start...), so maybe there is no need to add a `.gitpod.yml`.